pkg/staticpod/staticpod.go

```diff
@@ -231,6 +268,64 @@ func addVolumes(p *v1.Pod, src []string, dir bool) {
 	}
 }
 
+func addExtraMounts(p *v1.Pod, extraMounts []string) {
+	var (
+		prefix     = "extra-mount"
+		sourceType = v1.HostPathDirectoryOrCreate
+	)
+
+	for i, rawMount := range extraMounts {
+		mount := strings.Split(rawMount, ":")
+		var ro bool
+		switch len(mount) {
+		case 2:
```

Comment:
If we intend to keep this, can you add a comment as to why it's empty? Otherwise, remove it.